mailers/application_mailer.rb

```diff
@@ -0,0 +1,20 @@
+class ApplicationMailer
+  FROM_EMAIL = 'trainify@gmail.com'.freeze
+  
+  def self.send(*args)
+    new(*args).send
+  end
+
+  def send(params)
+    Thread.new do
+      Pony.mail(
+        to:      params[:email],
+        body:    params[:message],
+        subject: params[:subject],
+        from:    FROM_EMAIL,
+        via:     :sendmail,
+        headers: { 'Content-Type': 'text/html' }
+      )
+    end
+  end
```

Comment:
Why not something like this?

```ruby
class ApplicationMailer
  def self.send(*args)
    new(*args).send
  end

  def initialize(params = {})
    @params = params
  end

  def send
    Thread.new { Pony.mail(payload) }
  end

  protected

  def mailing_params
    raise NotImplementedError, 'You have to implement this method in the subclass'
  end

  private
  
  def payload
    defaults.merge(mailing_params)
  end

  attr_reader :params

  def defaults
    {
      from:    FROM_EMAIL,
      via:     :sendmail,
      headers: {
        'Content-Type': 'text/html'
      }
    }
  end
end
```

This'll allow you to subclass easily by defining only `payload` method